### PR TITLE
Updates for RELEASE.2023-07-18T17-49-40Z

### PR DESCRIPTION
**Pull request by djwfyi**
- Adds information about 3 attempts at site or bucket replication tasks
- Adds note about additional info for delete markers
- Updates mc admin info command example output

Partially addresses #931 .

Not staged, but can do if needed.

### CODE REVIEW

File: source/administration/bucket-replication.rst [hunk 1]
```diff
@@ -244,8 +240,12 @@ workers operating on that queue. MinIO continuously works to replicate and
...
 add to the queue. 
 
-MinIO queues failed replication operations and retries those operations until replication succeeds.
-This helps keep replication up-to-date without relying on the MinIO scanner to notice unreplicated object versions.
+
+.. versionchanged:: RELEASE.2022-07-18T17-49-40Z
+
+   MinIO queues failed replication operations and retries those operations up to three (3) times.
+   
+   Replication operations that fail more than three times wait for the scanner to identify the object for replication before trying again.
```
Comment (ravindk89) [inline source/administration/bucket-replication.rst:248]: This _might_ put too much agency on the operation itself.

Maybe

```suggestion
MinIO dequeues operations which fail to replicate after three attempts.
The scanner can pick up those affected objects at a later time and requeue them for replication.
```

File: source/operations/install-deploy-manage/multi-site-replication.rst [hunk 2]
```diff
@@ -88,6 +88,11 @@ Site Healing
 
 Any MinIO deployment in the site replication configuration can resynchronize damaged :ref:`replica-eligible data <minio-site-replication-what-replicates>` from the peer with the most updated ("latest") version of that data.
 
+.. versionchanged:: RELEASE.2023-07-18T17-49-40Z
+
+   Site replication operations retry up to three (3) times.
+   Failed operations wait for the MinIO Scanner to identify the replication task again before requeueing for another three attempts.
```
Comment (ravindk89) [inline source/operations/install-deploy-manage/multi-site-replication.rst:94]: Same general idea about putting the onus on the scanner vs an operation (e.g. PUT or GET) that waits.

File: source/reference/minio-mc-admin/mc-admin-info.rst [hunk 3]
```diff
@@ -21,21 +21,25 @@ for each MinIO server in the deployment.
 
 .. end-mc-admin-info-desc
 
+.. versionchanged:: RELEASE.2023-07-18T17-49-40Z
+
+   MinIO now reports the number of delete markers on the deployment.
```
Comment (ravindk89) [inline source/reference/minio-mc-admin/mc-admin-info.rst:26]:
```suggestion
   The output includes the number of delete markers on the deployment.
```

Kind of awkward to have this here - we don't make comments for when anything else was added.

Maybe add it after the example instead?

**Review by ravindk89 — APPROVED** [review]
I have some nits, but they are largely optional. LGTM with or without

**Review by feorlen — APPROVED** [review]
LGTM
Ravind already suggested a couple things I was thinking about, leave that up to you

File: source/administration/bucket-replication.rst (hunk 1 above)
Comment (djwfyi) [reply source/administration/bucket-replication.rst:248]: 👍

File: source/operations/install-deploy-manage/multi-site-replication.rst (hunk 2 above)
Comment (djwfyi) [reply source/operations/install-deploy-manage/multi-site-replication.rst:94]: Duplicated the updated warning from the bucket replication page.

File: source/reference/minio-mc-admin/mc-admin-info.rst (hunk 3 above)
Comment (djwfyi) [reply source/reference/minio-mc-admin/mc-admin-info.rst:26]: Just deleting it. The updated output includes delete marker count now. That's good enough.

**Pull request merged**